tests/groups/test_data.py

```diff
@@ -34,6 +33,26 @@ async def test_create(
     assert doc["_id"] == group.id == row.legacy_id
 
 
+class TestGet:
+    async def test_id_is_str(self, groups_data: GroupsData, fake2, snapshot):
+        fake_group = await fake2.groups.create()
+
+        await fake2.users.create(groups=[fake_group])
+
+        group = await groups_data.get(fake_group.id)
+
+        assert group == snapshot
+
+    async def test_id_is_int(self, groups_data: GroupsData, fake2, snapshot):
```

Comment:
```suggestion
    async def test_single_user(self, groups_data: GroupsData, fake2, snapshot):
```
Since we are working towards postgres first support for groups an integer id should be treated as the default case